### DISCUSSION TITLE
Reorder field columns on the project

### DISCUSSION THREAD

**Discussion by RickWinter** [community/community#13348] [Projects and Issues] [answered]
I can't find a way to change the order of the columns on the table.  Can you add a way to drag/drop the columns to update the view.

**Comment by MarkBennett** [comment]
It would be really nice if we could reverse the sort order when displaying a board with columns for each iteration. Otherwise the current iteration gets further and further to the right.

**Comment by SimonDarksideJ** [comment]
Agreed, we came to the agreement to add a new custom field to our GitHub project and needed it further up the list. However the order of custom fields is not editable it seems, except in table view

**Comment by Jay-o-Way** [comment]
@RickWinter dragging colums (left/right) in the table view is already possible. See #4803
@SimonDarksideJ sounds like you have something different in mind? Do you mean the order of fields as they are visible in the issue? #10044
![image](https://user-images.githubusercontent.com/65828559/210382812-fa62b518-cce7-4ee0-9c4e-3d9772f13739.png)

**Reply by SimonDarksideJ** [reply]
Not the filter but the fields.  So If I add a new custom field, I want it to appear higher in the form and not stuck to the last field @Jay-o-Way

**Reply by Jay-o-Way** [reply]
What do you mean by "filter"? What I have in mind is that I want "Priority" to be visible higher up the list. The issue that I linked deals with that idea.

**Reply by fcollonval** [reply]
I think I get the same error. The [documentation](https://docs.github.com/en/issues/planning-and-tracking-with-projects/customizing-views-in-your-project/customizing-the-table-layout#reordering-fields) says that the fields (aka the columns within a table view) can be reordered by drag. But it does not work (tested on Chromium and Firefox).

**Comment by choralonef9** [comment]
How do I rearrange columns in project?
To Move a column, click on the column name cell from the table header to select the entire column, hold down the column, drag it and release it in front of the column where you want to place it.

**Reply by fcollonval** [reply]
I tried again and it worked - don't know what was wrong earlier.

**Comment by tayloramurphy** [comment]
I don't know if the scope of this discussion is reflected in the title accurately. What I'm wanting to reorder is the Custom Fields list. The order in the Settings page of a Project is how it appears in an issue sidebar and there seems to be no way to update the list without deleting everything and starting over. 

We just added "Ending iteration" but because this was done after a few iterations of our process it's now completely disconnected from the original "Iteration" and we have no way to bring the two together.

<img width="308" alt="Screenshot 2023-02-14 at 4 02 42 PM" src="https://user-images.githubusercontent.com/5394188/218873491-b4cb91f4-1536-44b0-b3ec-486bbd9fde93.png">

**Reply by mlshapiro** [reply]
I have the same issue with a start/end date! I'd love to be able to drag the custom fields into the order they are shown in new issues/project items

**Reply by Jay-o-Way** [reply]
https://github.com/orgs/community/discussions/10044

**Reply by tayloramurphy** [reply]
Thank you for the link @Jay-o-Way

**Comment by rileybroughten** [comment, marked as answer]
You can now reorder fields from the settings page, which will reflect in the item side panel and from the issues page in the "Projects" section. Click to drag and drop the fields in the list. 

Check out the [changelog announcement](https://github.blog/changelog/2023-07-20-github-issues-projects-july-20th-update/#%F0%9F%A4%B8-reorder-fields-in-settings) for more details!

**Reply by tayloramurphy** [reply]
I said it on the other discussion, but thanks for shipping this! Huge QOL improvement!

**Comment by nyck33** [comment]
Can I reorder the columns? I'm trying to replicate Trello's Kanban board:

![image](https://github.com/community/community/assets/25189545/e75da9ee-5a85-43be-831b-a3ec5daeaf7c)

**Reply by nyck33** [reply]
Managed by deleting the default cards and making my own:

![image](https://github.com/community/community/assets/25189545/ce5fe97d-3c26-4c15-82c1-bc8712e01823)

**Reply by rileybroughten** [reply]
You can reorder the field values from the project settings page, such as reordering the default `Status` values which determines the order of the columns on your board 👍

**Comment by DevinSRose** [comment]
I tried reordering via drag and drop the columns in the Board view of my project. It highlighted the new location via a blue vertical bar when I hovered over the drop location, but then when I dropped it, nothing happened. This was on the Brave browser on Mac. 

I tried it on Google Chrome and the same operation worked, so fyi.
